assets/samba.yaml

```diff
@@ -0,0 +1,47 @@
+---
+- type: model
+  name: SambaLingo
+  organization: Samba Nova Systems
+  description: SambaLingo is a suite of models that adapt Llama 2 to a diverse set of 9 languages.
+  created_date: 2024-02-26
+  url: https://sambanova.ai/blog/sambalingo-open-source-language-experts
+  model_card: 
+    explanation: The Arabic language model card is given, but there exist one for each of the other 8 languages in the collection.
+    value: https://huggingface.co/sambanovasystems/SambaLingo-Arabic-Base
+  modality: text; text
+  analysis: Evaluated on open source multilingual model benchmarks.
+  size: unknown
+  dependencies: [LLaMA 2]
```

Comment:
I think this should be typeset as Llama 2 everywhere?

Reply:
Currently the original Llama is typeset as LLaMA everywhere, should I change those as well?